front/lib/api/assistant/actions/process.ts

```diff
@@ -399,7 +413,7 @@ export async function* runProcess(
       }
 
       if (event.content.block_name === "OUTPUT" && e.value) {
-        outputs = e.value as unknown[];
+        outputs = e.value as ProcessActionOutputsType;
```

Comment:
in an ideal world we'd ioTS that thing

Reply:
We don't do it anywhere except [here](https://github.com/dust-tt/dust/blob/090806b3ed62c10dac204ec5153693fa09c50f18/front/lib/actions/helpers.ts#L41-L109)

 But agreed, out of scope for this PR and not hugely necessary